src/Components/Organisms/Footer/Footer.tsx

```diff
@@ -0,0 +1,108 @@
+import { Container, Grid, Link, Stack } from '@mui/material';
+
+const Footer = () => {
+  const mediaXM = {
+    '@media screen and (max-width: 600px)': {
+      fontSize: '0.875rem',
```

Comment:
For links (according to figma) links should have font-size 16px <600px and 20px above.

Reply:
changed